src/composables/PluginManager.js

```diff
@@ -280,3 +281,21 @@ export async function addNewTemplateComponent(
 
   plugin.parse(config, fileInputs);
 }
+
+/**
+ * Get associated Monaco language key from file path.
+ * It will take in priority the language key from the plugin if it exists.
+ * Otherwise, it takes the language from the default languages of monaco.
+ *
+ * @param {String} path - File path.
+ * @return {string|null} Monaco language key.
+ */
+export function getLanguage(path) {
```

Comment:
don't think it's relevant to put it here.
only used  in one place & more about editor than plugin.

Reply:
done